lib/tasks/db.rake

```diff
@@ -28,7 +28,8 @@ ActiveRecord::Tasks::PostgreSQLDatabaseTasks.prepend(Module.new do
     clear_active_connections!
     establish_master_connection
     server_version = connection.select_value("SHOW server_version").to_i
-    suffix = server_version < 11 ? '' : '-12'
+    raise "PostgreSQL #{server_version} is not supported" if server_version < 10 || server_version == 11
+    suffix = server_version == 10 ? '-10' : ''
```

Comment:
why mark 11 as unsupported? I would suggest to leave it supported until we drop support for things below 12 altogether. But I'm fine if you feel confident nobody will suffer.

Reply:
The explanation is in the PR description - we never really officially supported v11, in fact, due to config differences, it will not work with the existing .sql files (we'd need to create a `structure-11.sql`)